specs/Deployment/DeploymentAPI.md

```diff
@@ -246,12 +264,164 @@ Here's the previous example extended with the `Show UI on error` option:
     }
 ```
 
+## Repair
+
+This API will always attempt to repair WindowsAppRuntime regardless of it's state. For that reason,
+this API should be called only when deployment status is not OK to avoid an unecessary repair even
+when WindowsAppRuntime is in good state. All information about the version, channel, and
+architecture needed are derived from the current WinAppSDK Framework package.
+
+Since both the Main _and_ the Singleton packages will be repaired, it is possible that Main package
+repair has failed and Initialize returns with PackageRepairFailed status (OR) it is possible that
```

Comment:
Here, "Initialize returns with PackageRepairFailed status" should actually be "Repair returns with PackageRepairFailed status".